source/getting-started/install.rst

```diff
@@ -90,6 +90,8 @@ Additional dependencies may be required depending on the host OS.
 
             `OrbStack <https://orbstack.dev/>`__ for **Mac** is supported by Exegol wrapper from ``v4.2.0``.
 
+            Your exegol installation cannot be stored under ``/opt`` directory when using OrbStack.
```

Comment:
Do you mind adding "(due to OrbStack limitations)"?
If an issue exists on OrbStack's repo, adding it as well could be relevant.

Reply:
Yes, I have added it here: <https://github.com/ThePorgs/Exegol-docs/pull/42/commits/339950b5cc86026e1234c152e10a1f200be10871>